packages/toolkit/src/lib/use-instill-form/components/component-output/MDTextViewer.tsx

```diff
@@ -77,14 +76,10 @@ export const MDTextViewer = ({
           </div>
         </div>
         <div>
-          <div
-            className={cn(
-              "markdown-body w-full overflow-x-scroll rounded-b-sm px-1.5 py-1",
-              enableFormattedText ? "" : "hidden",
-            )}
-          >
-            <Markdown>{sanitizedHtmlText}</Markdown>
-          </div>
+          <MarkdownViewer
+            className={enableFormattedText ? "" : "hidden"}
+            markdown={text ?? ""}
```

Comment:
Shouldn't this use `sanitizedHtmlText`?

Reply:
The MarkdownView has sanitized it, so I leave it there, but on the second though we can completely remove it in this layer